src/app/modules/shared-components/components/progress-section/boolean-section/boolean-section.component.html

```diff
@@ -1,38 +1,29 @@
-<alg-progress-section
+<alg-small-section
```

Comment:
boolean-section should be moved out of progress-section folder

Reply:
some important changes have been made, see [here](https://github.com/France-ioi/AlgoreaFrontend/pull/699#issuecomment-932177628)